pkg/runtime/runtime_test.go

```diff
@@ -1754,44 +1754,67 @@ func TestOnNewPublishedMessageGRPC(t *testing.T) {
 		Metadata: map[string]string{pubsubName: TestPubsubName},
 	}
 
+	envelope = pubsub.NewCloudEventsEnvelope("", "", pubsub.DefaultCloudEventType, "", topic, TestSecondPubsubName, "application/octet-stream", []byte{0x1}, "")
```

Comment:
If you know a better way to actually validate the payload here, I'm all ears. This test at least exercises the code, but I can't find a good way to validate the payload.

Reply:
It is a problem with this test.

Your code LGTM.